packages/auth-types/index.d.ts

```diff
@@ -266,6 +281,7 @@ export class FirebaseAuth {
     email: string,
     password: string
   ): Promise<any>;
+  signInWithEmailLink(email: string, emailLink: string): Promise<any>;
```

Comment:
`emailLink` is optional here. Change to:
`signInWithEmailLink(email: string, emailLink?: string): Promise<any>;`